packages/mds-audit/api.ts

```diff
@@ -430,25 +420,21 @@ function api(app: express.Express): express.Express {
           const { audit_event_id = uuid(), timestamp, telemetry } = req.body
 
           // Validate input params
-          if (
-            isValidAuditEventId(audit_event_id) &&
-            isValidTimestamp(timestamp) &&
-            isValidTelemetry(telemetry, { required: false })
-          ) {
-            // Create the audit end event
-            await writeAuditEvent({
-              audit_trip_id,
-              audit_event_id,
-              audit_subject_id,
-              audit_event_type: AUDIT_EVENT_TYPES.end,
-              ...flattenTelemetry(telemetry),
-              timestamp,
-              recorded
-            })
+          validateAuditApiAuditEndRequest({ audit_event_id, timestamp, telemetry })
```

Comment:
It could be good to consistently use one of `isValid` and `validate` when they're essentially serving the same purpose in the code; I _think_ that the `validate` method can throw errors, but `isValid` won't, and will instead return a `boolean`.